scaffolding-chef-infra/lib/scaffolding.sh

```diff
@@ -75,29 +75,31 @@ CFG_SPLAY_FIRST_RUN={{cfg.splay_first_run}}
 CFG_SPLAY_FIRST_RUN="\${CFG_SPLAY_FIRST_RUN:-0}"
 CFG_SSL_VERIFY_MODE={{cfg.ssl_verify_mode}}
 CFG_SSL_VERIFY_MODE="\${CFG_SSL_VERIFY_MODE:-:verify_peer}"
+CFG_CHEF_LICENSE={{cfg.chef_license.acceptance}}
+CFG_CHEF_LICENSE="\${CFG_CHEF_LICENSE:-undefined}"
 
 chef_client_cmd()
 {
-  chef-client -z -l \$CFG_LOG_LEVEL -c $pkg_svc_config_path/client-config.rb -j $pkg_svc_config_path/attributes.json --once --no-fork --run-lock-timeout \$CFG_RUN_LOCK_TIMEOUT
+  chef-client -z -l \${CFG_LOG_LEVEL} -c ${pkg_svc_config_path}/client-config.rb -j ${pkg_svc_config_path}/attributes.json --once --no-fork --run-lock-timeout \${CFG_RUN_LOCK_TIMEOUT} --chef-license \${CFG_CHEF_LICENSE}
```

Comment:
I'm not sure I understand why we are only using the CHEF_LICENSE_CMD in the windows scaffolding and not in the Linux scaffolding as well. I think by having this in the Linux scaffolding we can support an older version of the chef-client.